frontend/src/pages/EventPage/CreatePhysicalEventPage.js

```diff
@@ -0,0 +1,188 @@
+import React, { useReducer } from "react";
+import { useAuth } from "../../auth/useAuth";
+
+import {TextField, Typography, Stack} from "@mui/material";
+import GenericCardLayout from "../../layouts/GenericCardLayout";
+import ImageUploader from '../../components/ImageUploader';
+import LoadingButton from '../../components/LoadingButton';
+import MapSelectComponent from "../../components/MapSelectComponent"
+
+function CreatePhysicalEventForm() {
+
+  const [formInput, setFormInput] = useReducer(
+    (state, newState) => ({ ...state, ...newState }),
+    {
+      title: "",
+      description: "",
+      category: "",
+      eventPrice: "",
+      labels: "",
+      rules: "",
+      address: ""
+    }
+  );
+  const [selectedImage, setSelectedImage] = React.useState(null);
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [position, setPosition] = React.useState({lat: 41, lng: 29});
+
+
+  const formatFormInput = (posterId) => {
```

Comment:
Again if we need validation we should modify this component?

Reply:
I think we don't need to do validation in this form. I only updated the loading button to disable the button unless there is an image and I also made sure that the `eventPrice` is a numerical value.